packages/ui-app/src/AddressSummary.tsx

```diff
@@ -137,10 +139,19 @@ class AddressSummary extends React.PureComponent<Props> {
   }
 
   protected renderBalance () {
-    const { accounts_idAndIndex = [], balance, t, value, withBalance = true } = this.props;
+    const { accounts_idAndIndex = [], balance, t, value, withBalance = true, assetId = 'CENTRAPAY-T' } = this.props;
     const [_accountId] = accounts_idAndIndex;
     const accountId = _accountId || value;
 
+    let label;
+    if (assetId === 'CENTRAPAY-T') {
```

Comment:
it is ok to display `CENTRAPAY-T` or `CENNZ-T`. good to inform user they are test token.

Reply:
Updated.